tests/dynamic_data/publications/layer_raster_contrast/contrast_test.py

```diff
@@ -118,13 +142,20 @@ class TestLayer(base_test.TestSingleRestPublication):
     def test_contrast(layer: Publication, key, params, rest_method):
         """Parametrized using pytest_generate_tests"""
         base_file_name = key
-        file_name = f"{base_file_name}.{base_file_name.split('_')[0]}"
-        file_path = os.path.join(DIRECTORY, file_name)
-        assert_input_file(file_path, TEST_CASES[key]['expected_input'])
-        layer_params = {
-            'file_paths': [file_path],
-        }
-        rest_method(layer, params=layer_params)
+        test_case_def = deepcopy(TEST_CASES[key])
```

Comment:
You can prepare the whole rest_input before the test as part of the `test_cases` generation.